spark/build.sbt

```diff
@@ -18,10 +15,10 @@ libraryDependencies ++= Seq(
   scalatest % "test")
```

Comment:
Does spark project still require `awsSdkS3` dep? One of the motivations was to remove it to minimize version conflicts if our s3 module is not used.